.circleci/config.yml

```diff
@@ -514,7 +530,15 @@ workflows:
             branches:
               only:
                 - '/^release-0\.\d+$/'
-
+      - release_bom:
+          context:
+            - github-repo
+            - gcp-astronomer-prod
+          filters:
+              branches:
+                only:
+                  - '/^release-0\.\d+$/'
```

Comment:
We should do this on tag, not branch. Here's an example of how it works with tags: https://github.com/astronomer/astro/blob/e071bd49ae/.circleci/config.yml#L427-L431

We need to find a regex that matches our tag pattern, which would be `v<semver-compatible-regex>`